src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/OrderReturnStates/Blocks/form.html.twig

```diff
@@ -40,8 +40,8 @@
             'label': 'Status name'|trans({}, 'Admin.Shopparameters.Feature'),
             'help': 'Order\'s return status name.'|trans({}, 'Admin.Shopparameters.Help')
                     ~ ' '
-                    ~ 'Invalid characters: numbers and'|trans({}, 'Admin.Shopparameters.Help')
-                    ~ ' !<>,;?=+()@#"{}_$%:'
+                    ~ 'Invalid characters:'|trans({}, 'Admin.Shopparameters.Help')
```

Comment:
I know it's not the scope of the PR, but this form is a bit "ol school" it uses the helpers `ps.form_group_row` and stuff All this wording should be configured in the Symfony form type and the twig template should be as simple as `form_row(orderReturnStateForm)`

If you prefer to handle this in another PR though no problem